cmd/init.go

```diff
@@ -53,24 +53,22 @@ func (c *InitCommand) Run(args []string) int {
 		return 1
 	}
 
-	c.UI.Info("✓ Virtualenv created\n")
+	c.UI.Info(color.GreenString("✓ Virtualenv created"))
 
-	pip := exec.Command("pip", "install", "-r", "requirements.txt")
+	pip := CommandExecWithStderrOnly("pip", []string{"install", "-r", "requirements.txt"}, c.UI)
```

Comment:
What about copying how `galaxy install` works?

https://github.com/roots/trellis-cli/blob/f17dd81ec3320559b56bdb98b05f7b7b9f3aff8a/cmd/galaxy_install.go#L53-L64

Then we don't need the new `CommandExecWithStderrOnly` and you have more control over the output to add the newlines?

Reply:
The goal of `CommandExecWithStderrOnly` is to retain coloring of the wanring messages. Here is a screenshot of `CommandExecWithStderrOnly` vs the `galaxy install` way:

![trellis-init-coloring](https://user-images.githubusercontent.com/2259834/100555622-89993580-3294-11eb-8325-0b8113d01b87.png)

Reply:
Weird, I tried it out and got the same. Found this https://stackoverflow.com/questions/61113201/io-multiwriter-removes-terminal-colors-from-output but it wasn't really any help.

Okay, interesting I think this is why https://stackoverflow.com/a/29375330

`pip` only has a `--no-color` option but not way to force it. Who knows if there's actually a solution so what you have works 👍